geomesa-jupyter/src/main/scala/org/locationtech/geomesa/jupyter/Leaflet.scala

```diff
@@ -31,6 +40,52 @@ object L {
        """.stripMargin
   }
 
+  /**
+    * Takes a string containing a valid javascript styling function as its argument
+    * @param javascriptFunction The javascript styling function
+    */
+  case class StyleOptionFunction(javascriptFunction: String) extends StyleOption {
+    def render: String = javascriptFunction
```

Comment:
This is great! Can you add some docs here or on the ``README.md`` on how to use it? Specifically, what will be passed to the function, and what should it return?